lib/core/view_models/login_view_model.dart

```diff
@@ -1,54 +1,30 @@
-import 'package:flutter/material.dart';
 import 'package:pedantic/pedantic.dart';
 import 'package:provider_start/core/constant/view_routes.dart';
 import 'package:provider_start/core/enums/view_state.dart';
 import 'package:provider_start/core/exceptions/auth_exception.dart';
 import 'package:provider_start/core/mixins/validators.dart';
 import 'package:provider_start/core/services/auth/auth_service.dart';
 import 'package:provider_start/core/services/navigation/navigation_service.dart';
+import 'package:provider_start/core/utils/simple_log_printer.dart';
 import 'package:provider_start/core/view_models/base_view_model.dart';
 import 'package:provider_start/locator.dart';
 
 class LoginViewModel extends BaseViewModel with Validators {
-  final _navigationService = locator<NavigationService>();
+  final _log = getLogger();
```

Comment:
This is the wrong logger. Use the static logger client from `import 'package:provider_start/core/utils/logger.dart';`